packages/blockchain-wallet-v4-frontend/src/modals/Interest/InterestForm/selectors.ts

```diff
@@ -2,13 +2,15 @@ import { lift } from 'ramda'
 import { selectors } from 'data'
 
 export const getData = state => {
-  const coin = selectors.components.borrow.getCoinType(state)
+  const coin = selectors.components.interest.getCoinType(state)
   const ratesR = selectors.components.borrow.getRates(state)
   const supportedCoinsR = selectors.core.walletOptions.getSupportedCoins(state)
+  const interestLimitsR = selectors.components.interest.getInterestLimits(state)
 
-  return lift((rates, supportedCoins) => ({
+  return lift((rates, supportedCoins, interestLimits) => ({
     coin,
+    minimumDeposit: interestLimits.BTC.minimumDeposit,
```

Comment:
Probably don't want to hardcode BTC here, I'd also recommend just passing the enter interestLimits object to the component